htdocs/reception/class/api_receptions.class.php

```diff
@@ -123,6 +123,7 @@ public function index($sortfield = "t.rowid", $sortorder = 'ASC', $limit = 100,
 		$sql = "SELECT t.rowid";
 		$sql .= " FROM ".MAIN_DB_PREFIX."reception AS t";
 		$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."reception_extrafields AS ef ON (ef.fk_object = t.rowid)"; // Modification VMR Global Solutions to include extrafields as search parameters in the API GET call, so we will be able to filter on extrafields
+		$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."element_element as el ON el.fk_target = t.rowid AND el.targettype = 'reception' AND el.sourcetype = 'order_supplier'";
```

Comment:
If not filter (el.fk_origin:=:123) is provided, this will generate a duplication of lines
So this must be conditionned on having a filter on el field.